hs-spanshot/test/CaptureStreamSpec.hs

```diff
@@ -0,0 +1,160 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
This file should also test that the boundaries (e.g too early events should not be consider and too late events)? Is it covered somewhere else?